Maple2.Server.Game/PacketHandlers/ItemUseHandler.cs

```diff
@@ -72,6 +72,13 @@ public class ItemUseHandler : PacketHandler<GameSession> {
             case ItemFunction.VIPCoupon:
                 HandlePremiumClubCoupon(session, item);
                 break;
+            case ItemFunction.SelectItemBox:
+                HandleSelectItemBox(session, packet, item);
+                break;
+            case ItemFunction.OpenItemBox:
```

Comment:
`OpenItemBoxLullu` and `OpenItemBoxLulluSimple` can probably be included as well.

Reply:
I was gonna include this in a separate PR, along with ducky capsules and style crates